db/migrate/20160415121724_create_cloud_services.rb

```diff
@@ -0,0 +1,21 @@
+class CreateCloudServices < ActiveRecord::Migration[5.0]
+  def change
+    create_table :cloud_services do |t|
+      t.string :ems_ref
+      t.string :source
+      t.string :binary
+      t.string :hostname
```

Comment:
Remove `hostname`.  This should be accessible via `cloud_service.host.hostname`.

We don't want to have the extra housekeeping of updating this table if the `host`'s `hostname` changes for some reason during inventory refresh.